controllers/runner_pod_controller.go

```diff
@@ -429,3 +431,20 @@ func (r *RunnerPodReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Named(name).
 		Complete(r)
 }
+
+func (r *RunnerPodReconciler) updateCachedRunners(ctx context.Context, enterprise, org, repo string) error {
+	now := time.Now().Add(-60 * time.Second)
+	if !r.RunnerCache.ExpirationTime.Time.IsZero() && now.Before(r.RunnerCache.ExpirationTime.Time) {
+		return nil
+	}
+	// if context of time is not passed and the cached_runners is not null, return cached runners
+	// else, call the runners and 'cache' it to return
+	runners, err := r.GitHubClient.ListRunners(ctx, enterprise, org, repo)
+	// always cache runners for one minute regardless of other caching in GitHub
+	r.RunnerCache = RunnerCache{
+		ExpirationTime: metav1.Time{Time: time.Now().Add(60 * time.Second)},
```

Comment:
I would make this a constant or a config item instead of hardcoding it to 1 minute

Reply:
Agreed. The goal was to at least reduce the GitHub API calls down to (at the most) 60 per minute per reconciler.  I figured it was a good approach to get it set to 1m in the MVP, then work on the configurable asset after - but we can definitely make it a `CONST` until then